nautobot/dcim/models/device_component_templates.py

```diff
@@ -465,54 +465,25 @@ def clean(self):
 
 
 @extras_features("custom_validators")
```

Comment:
Do our usual other features (custom_links, export_templates, graphql, webhooks) not apply to ModuleBayTemplate? Or is it just that this is the pattern we have for all the other ComponentTemplateModels and something we should fix across the board?

Reply:
Ah right - we don't have detail-views for these models, do we? So custom_links at least wouldn't apply. I think the others would though. I'll open an issue.

Reply:
#5789